Tests/IntegrationTests.Shared/PrimaryKeyTests.cs

```diff
@@ -275,6 +275,27 @@ public void PrimaryKeyFailsIfClassNotinRealm()
                 var obj = skinny.ObjectForPrimaryKey<PrimaryKeyInt64Object>(42);
             });
         }
+
+        // mostly a little test to show iterating schema and getting primary key names
+        [Test]
+        public void CanGetNameOfPrimaryKeyFromSchema()
+        {
+            int classesWithPK = 0;
+            int classesWitoutPK = 0;
+            foreach (var objSchema in _realm.Schema)
+            {
+                var primaryKeyProp = objSchema.Where(prop => prop.IsPrimaryKey).FirstOrDefault();
```

Comment:
We could simplify by using `FirstOrDefault(prop => prop.IsPrimaryKey)`, no need for a `.Where`.